packages/security/__tests__/CredentialManagerFactory.test.ts

```diff
@@ -138,23 +139,6 @@ describe("CredentialManagerFactory", () => {
             );
         });
 
-        it("should handle being passed an object that isn't a class or string", async () => {
```

Comment:
should we keep this test instead of removing it?
Not sure what exactly this was testing.

Maybe that the `initialize` function was able to set the invalid credential manager if it failed to get some common Manager methods (like `save`)

Reply:
It tested when an object was passed into the imperative override (when before only strings and class were supported), I removed this one since it checks for a failure to load an object which shouldn't fail with this new addition